lib/corpus.py

```diff
@@ -25,6 +25,9 @@ def __init__(self):
         super().__init__()
         self.name: Optional[str] = None
 
+    def __repr__(self):
+        return f"{self.__class__.__name__}({self.fullname()})"
```

Comment:
There is the convention that either what you return by `__repr__` is an `eval`-able string, i.e. a valid Python expression which would result in the object. If the format is like `${classname}(${args})`, this is the case. However, even though it looks like valid Python, it's not correct here.

Or, if you don't intend to return a valid Python expression, then the convention is to print it deliberately not using valid Python syntax, e.g. like `<${classname} ${args}>`.

Reply:
I see, thanks for the hint! I'll wait for a conclusion in the other threads to resolve this.